src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -162,78 +164,63 @@ These events are also available for querying, creating custom dashboards, and al
   </tbody>
 </table>
 
-## Automatic Anomalies [#automatic-anomalies]
-
-Automatic anomaly detection uses the following methods to detect anomalies in your app data:
-
-1. Anomaly detection monitors metric data reported by an APM agent, building a model of your typical application dynamics, and focuses on key golden signals: throughput, response time, and errors.
-2. If one of these golden signals shows anomalous behavior, the system flags the behavior and tracks recovery to normal behavior.
-3. The system adapts to changes in your data, and continuously updates models based on new data.
-
-**Automatically on**: By default, anomaly detection monitors all your APM applications, with no action required by you. When an anomaly is detected, it's automatically surfaced in various activity streams, the applied intelligence anomalies feed and is available for querying via NRQL.
-
-**Correlation**: Anomalies are connected to our correlation logic, otherwise known as decisions. If our correlation logic determines that issues are connected, then a single issue will be created from multiple incidents. You can see this correlated issue in the issues feed. Anomalies provide additional context about what changed around the time an issue occurred so your team can easily detect patterns and diagnose problems. 
-
-**Receiving notifications:** When New Relic detects anomalous changes in throughput, error rate, or response time, your team has the option to receive notifications about this behavior. We recommend sending notifications selected Slack channels or email channels. When the anomaly goes back to normal then a recovery message is sent. If you don't want to receive notifications you can still access to the data using a [NRQL query](#anomaly-events).
+## Anomaly set-up [#set-up]
 
-**Anomaly analysis:** For each anomaly, we provide a link in Slack to an analyze anomaly page. This page generates automatic insights into the anomaly. The page is also available from the anomalies tab, which lists recent anomalies. This page uses your existing APM and anomaly detection data to provide explanations as to the cause of the anomaly.
+Once you choose to monitor anomalous behavior in your system using either our custom or automatic anomaly detection, you will need to make sure that your team is notified of any unusual behavior and that you can query and understand your data. It doesn't matter if you choose custom or automatic anomaly detection– the set-up is the same. 
```

Comment:
I've been steered away from using "you will", so I thought I would pass that along here as well.